common/client-core/config-types/src/lib.rs

```diff
@@ -711,6 +711,12 @@ pub struct DebugConfig {
 
     /// Defines all configuration options related to the forget me flag.
     pub forget_me: ForgetMe,
+
+    /// Indicates whether to mix hops or not. If mix hops are enabled, traffic
+    /// will be routed as usual, to the entry gateway, through three mix nodes, egressing
+    /// through the exit gateway. If mix hops are disabled, traffic will be routed directly
+    /// from the entry gateway to the exit gateway, bypassing the mix nodes.
+    pub disable_mix_hops: bool,
```

Comment:
I think it'd make more sense if it was part of `Traffic` section rather than being a generic setting. 
this would also give us explicit lack of control over cover traffic which would always send full packets

Reply:
Added to the `Traffic` config object